src/chrome/content/rules/Pride-Toronto.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Active mixed content:
+		- www.pridetoronto.com
+
+	Invalid certificate:
+		- register.pridetoronto.com
+
+	Timed out:
+		- pridetoronto.ca, equivalent to www.pridetoronto.com
+		- www.pridetoronto.ca, equivalent to www.pridetoronto.com
+-->
+
+<ruleset name="Pride Toronto" platform="mixedcontent">
+	<target host="pridetoronto.ca" />
+	<target host="www.pridetoronto.ca" />
```

Comment:
Looks like the `pridetoronto.ca` domain has been sold. Can you double check please?

Reply:
Looks like that's the case. Closing.